app/views/admin/governments/_form.html.erb

```diff
@@ -1,18 +1,90 @@
-<%= form_for [:admin, government], as: :government, html: {class: 'government-form'} do |government_form| %>
-  <%= government_form.errors %>
-  <p><%= government_form.text_field :name, {placeholder: "eg 2005 to 2010 Labour government"} %></p>
-  <p><%= government_form.text_field :start_date, {placeholder: "eg 2005-05-06"} %></p>
-  <p><%= government_form.text_field :end_date, {placeholder: "eg 2010-05-11"} %></p>
+<%= form_for [:admin, government], as: :government do |form| %>
+  <%= render "govuk_publishing_components/components/input", {
+    label: {
+      text: "Name"
+    },
+    name: "government[name]",
+    id: "government_name",
+    heading_level: 3,
+    heading_size: "m",
+    value: government.name,
+    error_message: errors_for_input(government.errors, :wont_be_empty)
```

Comment:
This needs to be 
```
error_message: errors_for_input(government.errors, :name)
```

The way this helper method works is that it takes a models errors, and the attribute.

<img width="1007" alt="image" src="https://user-images.githubusercontent.com/42515961/225645039-a1b1c8fa-d916-40f3-bcf2-06fdda13145c.png">

The code iterates through the errors  and grabs the errors where the attribute matches attribute argument passed in.